manifests/monitoring/sensu.pp

```diff
@@ -0,0 +1,18 @@
+# See README.md for details
+class consul_replicate::monitoring::sensu {
+
+  package { 'sensu':
+    ensure => installed
+  }
```

Comment:
Should there be a -> here?  Without ordering, it would be possible for the check-procs.rb check to fail if sensu isn't yet installed, right?